<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ohmetro</class>
 <widget class="QMainWindow" name="Ohmetro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ohmetro</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resistencia &gt; 1000 ohms</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resistencia &lt; 100 ohms</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="rOhms">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ohms</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rKOhms">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>KOhms</string>
      </property>
     </widget>
     <zorder>rOhms</zorder>
     <zorder>rKOhms</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblDown">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_3</zorder>
    <zorder>groupBox</zorder>
    <zorder>lcdNumber</zorder>
    <zorder>lblDown</zorder>
    <zorder>lblUp</zorder>
    <zorder>label_5</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
